plugins/woocommerce/package.json

```diff
@@ -38,7 +38,7 @@
 		"env:test": "pnpm env:dev",
 		"env:perf": "pnpm env:dev && pnpm env:performance-init",
 		"preinstall": "npx only-allow pnpm",
-		"postinstall": "XDEBUG_MODE=off composer install --quiet",
+		"postinstall": "rimraf -g vendor/woocommerce && XDEBUG_MODE=off composer install --quiet",
```

Comment:
As the packages are getting copied and we'll be jumping between symlink/copy, the composer does not always do what is expected. Hence, we are giving it a helping hand.